pages/admin/request/create-new.tsx

```diff
@@ -0,0 +1,845 @@
+            lastName: guardian.lastName,
+            phone: guardian.phone,
+            relationship: guardian.relationship,
+            addressLine1: guardian.addressLine1,
+            addressLine2: guardian.addressLine2,
+            city: guardian.city,
+            postalCode: guardian.postalCode,
+            poaFormUrl: '', //TODO
+          });
+        } else {
+          setGuardianInformation({
+            omitGuardianPoa: false,
+            firstName: '',
+            middleName: '',
+            lastName: '',
+            phone: '',
+            relationship: '',
+            addressLine1: '',
+            addressLine2: '',
+            city: '',
+            postalCode: '',
+            poaFormUrl: '', //TODO
+          });
+        }
+      }
+    },
+  });
+
+  /**
+   * Sets and fetches permit holder data when selected from typeahead
+   */
+  const handleSelectPermitHolder = useCallback(applicantId => {
+    setApplicantId(applicantId || null);
+    if (applicantId) {
+      getApplicant({
+        variables: {
+          id: applicantId,
+        },
+      });
+    }
+  }, []);
+
+  /**
+   * Submit application mutation
+   */
+  const [submitNewApplication, { loading: submitRequestLoading }] = useMutation<
+    CreateNewApplicationResponse,
+    CreateNewApplicationRequest
+  >(CREATE_NEW_APPLICATION_MUTATION, {
+    onCompleted: data => {
+      if (data) {
+        const { ok, applicationId } = data.createNewApplication;
+        if (ok) {
+          toast({
+            status: 'success',
+            description: 'New application has been submitted!',
+            isClosable: true,
+          });
+
+          if (applicationId) {
+            router.push(`/admin/request/${applicationId}`);
+          }
+        }
+      }
+    },
+    onError: error => {
+      toast({
+        status: 'error',
+        description: error.message,
+        isClosable: true,
+      });
+    },
+  });
+
+  /**
+   * Handle new APP request submission
+   */
+  const handleSubmit = async (event: SyntheticEvent) => {
+    event.preventDefault();
+
+    if (physicianAssessment.patientCondition === null) {
+      toast({ status: 'error', description: 'Missing patient condition', isClosable: true });
+      return;
+    }
+
+    if (physicianAssessment.permitType === null) {
+      toast({ status: 'error', description: 'Missing permit type', isClosable: true });
+      return;
+    }
+
+    if (additionalQuestions.usesAccessibleConvertedVan === null) {
+      toast({
+        status: 'error',
+        description: 'Missing if patient uses accessible converted van',
+        isClosable: true,
+      });
+      return;
+    }
+
+    if (additionalQuestions.requiresWiderParkingSpace === null) {
+      toast({
+        status: 'error',
+        description: 'Missing if patient requires wider parking space',
+        isClosable: true,
+      });
+      return;
+    }
+
+    if (doctorInformation.mspNumber === null) {
```

Comment:
```suggestion
    if (!doctorInformation.mspNumber) {
```
This will catch all empty values